CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+<a name="0.4.8"></a>
```

Comment:
**revert:** please do not add the release changes to your PR, there should be a separate chore commit just for that (I usually run it because I need to push the release git tag to the repository)

Reply:
tip: If just *you* run it, try to not keep it in the repo, If someone try to run all npm scripts to ensure 'all' things have done, comes to this.